### DISCUSSION TITLE
Fetch links default behaviour

### DISCUSSION THREAD

**Discussion by Vitalium** [BeanieODM/beanie#351] [Feature Request]
In my project I need to always fetch all links. I reviewed source code and noticed that fetch_link is set to False in kwargs everywhere. Of course, I can subclass Document and change signatures of all functions by changing fetch_link to True. But maybe it will be useful to set default action (True or False) somewhere is global settings/options?

**Comment by roman-right** [comment]
Sounds good. Yup, I'll add this to my todo
